src/Bridges/Constraint/quad_to_soc.jl

```diff
@@ -194,10 +194,44 @@ function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintPrimal,
     return output
 end
 
-# TODO
-#function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintFunction,
-#                 b::QuadtoSOCBridge)
-#end
+# Lemma: If (1, s, x), (v, u, y) in RotatedSecondOrderCone and
+#        (1, s, x) ⋅ (v, u, y) = 0, then ∃σ ≥ 0 that,
+#        y = -σ*x, u = σ, v = σ*||x||_2^2/2.
+# Proof: We have
+#        (1, s, x) ⋅ (v, u, y) = v + s * u + x ⋅ y
+#        (Cauchy-Schwarz)      ≥ v + s * u - ||x||_2 * ||y||_2
+#        (RotatedSOC)          ≥ v + s * u - 2 * √(s * u * v)
+#        (AM-GM)               ≥ v + s * u - 2 * (v + s * u) / 2
+#                              = 0
+#        By assumption, the left-hand side is zero, hence all inequalities
+#        are equalities. By Cauchy-Schwarz, this means that ∃σ ≥ 0 such
+#        that `y = -σ*x`. By AM-GM, we have `v = s * u`.
+#        By RotatedSOC, we have either:
+#        1) `||y||_2^2 < 2 * u * v` and `||x||_2^2 = 2 * s = 0`: That implies
+#           that `v = 0 * u = 0` hence `||y||_2^2 < 0` which is impossible.
+#        2) `||x||_2^2 < 2 * s` and `||y||_2^2 = 2 * u * v = 0`: we have either:
+#           a) `u = 0`: hence `v = s * u = 0` or
+#           b) `v = 0`: since `s > 0`, `u = v / s = 0`.
+#           In any case, `x = 0` and `u = v = 0` hence we can take `σ = 0`.
```

Comment:
It could be mentioned that `x = 0` follows from `||y|| = 0` and `y = -σ*x`.

Reply:
Oops, it should be `y = 0`